examples/blog/store-default/__index.nt

```diff
@@ -1,3 +1,18 @@
 <http://localhost:9000/> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://localhost:9000/api/schema/Blog> .
 <http://localhost:9000/> <http://www.w3.org/2000/01/rdf-schema#label> "My Blog" .
 <http://localhost:9000/> <http://localhost:9000/api/schema/post> <http://localhost:9000/post/1> .
+
+<http://localhost:9000/> <http://www.w3.org/ns/hydra/core#search> _:searchTemplate .
+_:searchTemplate <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/hydra/core#IriTemplate> .
+_:searchTemplate <http://www.w3.org/ns/hydra/core#template> "/{?from,to}" .
+_:searchTemplate <http://www.w3.org/ns/hydra/core#variableRepresentation> <http://www.w3.org/ns/hydra/core#BasicRepresentation> .
+
+_:searchTemplate <http://www.w3.org/ns/hydra/core#mapping> _:fromMapping .
+_:fromMapping <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/hydra/core#IriTemplateMapping> .
+_:fromMapping <http://www.w3.org/ns/hydra/core#variable> "from" .
+_:fromMapping <http://www.w3.org/ns/hydra/core#property> <http://localhost:9000/api/schema/from> .
+
+_:searchTemplate <http://www.w3.org/ns/hydra/core#mapping> _:toMapping .
+_:toMapping <http://www.w3.org/ns/hydra/core#property> <http://localhost:9000/api/schema/to> .
+_:toMapping <http://www.w3.org/ns/hydra/core#variable> "to" .
+_:toMapping <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/hydra/core#IriTemplateMapping> .
```

Comment:
I think I would generate this dynamically in the handler, adding to `req.hydra.resource.dataset`.

On the other hand we eventually want to have this loaded from the API Documentation on the client but that is not implemented.

What do you think?

Reply:
We can add the code to generate it later.